src/Anonymization/Anonymizer/Core/ConstantAnonymizer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core;
+
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\AbstractAnonymizer;
+use MakinaCorpus\DbToolsBundle\Attribute\AsAnonymizer;
+use MakinaCorpus\QueryBuilder\Query\Update;
+
+#[AsAnonymizer(
+    name: 'constant',
+    pack: 'core',
+    description: <<<TXT
+    Set all value to a constant value.
+    Options are:
+        - `value`: the value you want to use to fill the column
+        - `type`: a SQL type for the given value (default is 'text')
+    TXT
+)]
+class ConstantAnonymizer extends AbstractAnonymizer
+{
+    /**
+     * @inheritdoc
+     */
+    public function anonymize(Update $update): void
+    {
+        if (!$this->options->has('value')) {
+            throw new \InvalidArgumentException("'value' option should be given.");
+        }
+
+        $expr = $update->expression();
+
+        $update->set(
+            $this->columnName,
+            $this->getSetIfNotNullExpression(
+                $expr->cast(
```

Comment:
Not `->cast()`, but `->value()` here !

`cast()` will simply add the `CAST(? AS type)` SQL expression, whereas `value()` will additionally trigger the PHP to SQL value converter.

Reply:
And it could be nice to link the value converter documentation in the related anonymizer documentation page: https://php-query-builder.readthedocs.io/en/stable/converter/converter.html#input-value-converter